eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -0,0 +1,44 @@
+parameters:
+  PackageName: 'not-specified'
+  ServiceName: 'not-specified'
+  ForRelease: ''
+
+steps:
+  - pwsh: |
+      Import-Module "$(Build.SourcesDirectory)/eng/common/scripts/modules/common-manifest.psd1"
+      $Language = "$(Build.Repository.Name)".Substring("$(Build.Repository.Name)".LastIndexOf('-') + 1)
```

Comment:
This looks like it might be interesting to include in our common-manifest module or perhaps in another yml template that allows us to setup a common variable we can use for Language.